helm/chaos-mesh/Chart.yaml

```diff
@@ -1,12 +1,12 @@
 apiVersion: v2
-appVersion: "v1.2.0"
+appVersion: "v1.2.1"
 description: Chaos Mesh® is a cloud-native Chaos Engineering platform that orchestrates chaos on Kubernetes environments.
 home: https://chaos-mesh.org
 icon: https://raw.githubusercontent.com/pingcap/chaos-mesh/master/static/logo.svg
 sources:
   - https://github.com/pingcap/chaos-mesh
 name: chaos-mesh
-version: v0.5.0
+version: v0.5.1
```

Comment:
The definition of helm files has not updated, do we need to bump this version?

Reply:
This [PR](https://github.com/chaos-mesh/chaos-mesh/pull/1867) fixed an issue related to the helm files.

Reply:
Sorry I missed that https://github.com/chaos-mesh/chaos-mesh/pull/1867 which includes updates with the helm.